openwebwork/webwork2
# Add Contrib option to Library Browser

Now that Contrib is almost as large as the OPL itself, it seems time to support the searching of Contrib alongside the OPL in the Library Browser. 

Testing this PR requires the OPL release created by drdrew42/webwork-open-problem-library@c2eb5be26b7910167fb1b5afb58da0f5879a9d7f

To test:

Load up the OPL release [here](https://github.com/drdrew42/webwork-open-problem-library/releases/tag/vLib-test-5)

Go to Library Browser, select a DBsubject/chapter/section, and check the "Include Contrib" box. Problems from Contrib will appear at the end of search, so you might have to go through multiple pages (or increase the number of results shown per page).
